pep-0001.txt

```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
 The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate for that PEP, and they will then have the
+authority to approve (or reject) that PEP. 
+
+The term "PEP-Delegate" will be used under the Steering
+Council governance model. The PEP's designated decision maker,
+the "PEP-Delegate" with the Steering Council's support, is
+recorded in the "PEP-Delegate" field in the PEP's header. For
+PEPs written prior to the Steering Council's governance model, the field name,
+"BDFL-Delegate" will still be kept.
+The terms, PEP-Delegate (preferred term under the Steering Council governance 
+model) and BDFL-Delegate, may be used interchangeably in discussion.
```

Comment:
```suggestion
The terms PEP-Delegate and BDFL-Delegate may be used interchangeably in 
discussion. PEP-Delegate is the preferred term under the Steering Council 
governance model.
```

Reply:
I might restructure this a bit to mention PEP-Delegate first and BDFL-Delegate as a legacy alias that should be understood to mean the same thing, but it's not terribly important.